lib/parser.js

```diff
@@ -186,10 +194,16 @@ module.exports = function parse (line) {
         }
     }
 
-    // is existing variable
-    if (keys[1] === 'is') {
-        statement += keys[0] + ' = ';
-        if (keys[2] === 'new') {
+    // support any kind of assignment operator
+    var assignment = keys[1]
+    if(assignOps.hasOwnProperty(assignment))
+    {
+      statement += keys[0] + assignOps[assignment];
+      
+      // only support new for is, others don't make sense
```

Comment:
Is there a reason not to allow this?  Disallowing it seems odd and there is a potential that someone will want it to work

Reply:
Does it make sense to have: `something += new Something()` ? I didn't think of a use case.

Reply:
Technically that would compile to something like `something += (new Something).valueOf()` i believe, so we probably want to support this:
`something += new Number(5)`